example/demo/build.zig

```diff
@@ -0,0 +1,27 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) void {
+    const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
+    const exe = b.addExecutable(.{
+        .name = "exe",
+        .root_module = b.createModule(.{
+            .root_source_file = b.path("main.zig"),
+            .target = target,
+            .optimize = optimize,
+            .imports = &.{.{
+                .name = "renderer",
+                .module = b.createModule(.{
+                    .root_source_file = @import("ZTT").templateRenderer(
+                        b,
+                        b.dependency("ZTT", .{}),
+                        b.path("main.ztt"),
+                    ),
```

Comment:
I would put that much further above in the code, probably between `optimize` and `exe`, and make it visible. That's the part the user cares about and it's hard to see here.

Also please store the dependency in a separate variable, so it's easier to see :)